### PR TITLE
Add github action to release helm charts

### PR DESCRIPTION
**Pull request by kvaps**
Add github action to create chart releases automatically once the chart version is bumped.
After this PR and creation of orphan branch named `gh-pages` - this repository could be used in `helm repo add` command.

```
helm repo add nfs-ganesha-server-and-external-provisioner https://kubernetes-sigs.github.io/nfs-ganesha-server-and-external-provisioner/
```

For example, my fork can be used:

```
$ helm repo add nfs-ganesha-server-and-external-provisioner https://kvaps.github.io/nfs-ganesha-server-and-external-provisioner/
"nfs-ganesha-server-and-external-provisioner" has been added to your repositories
$ helm search repo
NAME                                           	CHART VERSION	APP VERSION	DESCRIPTION                                       
nfs-ganesha-server-and-external-provisioner/nfs...	1.3.2             	3.0.0             	nfs-server-provisioner is an out-of-tree dynami...
```

To create the `gh-pages` branch use: `git checkout --orphan gh-pages`.

### CODE REVIEW

File: charts/nfs-server-provisioner/README.md [hunk 1]
```diff
@@ -29,7 +29,8 @@ package manager.
 To install the chart with the release name `my-release`:
 
 ```console
-$ helm install stable/nfs-server-provisioner --name my-release
+$ helm repo add nfs-ganesha-server-and-external-provisioner https://kubernetes-sigs.github.io/nfs-ganesha-server-and-external-provisioner/
```
Comment (consideRatio) [inline charts/nfs-server-provisioner/README.md:32]: Perhaps `helm repo update` required here also?

Comment (consideRatio) [reply charts/nfs-server-provisioner/README.md:32]: I've stopped doing this and rely on `--repo` flags when doing `helm install` or `helm upgrade` so I've forgotten.

**Review by consideRatio — APPROVED** [review]
Besides a documentation comment, this LGTM!

**Comment by kmova** [comment]
@kvaps - the `gh-pages` branch is pushed. I am good with this PR. Would you like me to go ahead or wait for resolving the minor doc comment.

**Pull request merged**